--- linux/sound/drivers/aloop.c.orig	2021-04-13 20:06:31.972047456 +0200
+++ linux/sound/drivers/aloop.c	2021-04-13 20:13:01.432132431 +0200
@@ -552,9 +552,9 @@
 			 SNDRV_PCM_FMTBIT_S24_3LE | SNDRV_PCM_FMTBIT_S24_3BE |
 			 SNDRV_PCM_FMTBIT_S32_LE | SNDRV_PCM_FMTBIT_S32_BE |
 			 SNDRV_PCM_FMTBIT_FLOAT_LE | SNDRV_PCM_FMTBIT_FLOAT_BE),
-	.rates =	SNDRV_PCM_RATE_CONTINUOUS | SNDRV_PCM_RATE_8000_192000,
+	.rates =	SNDRV_PCM_RATE_CONTINUOUS | SNDRV_PCM_RATE_8000_384000,
 	.rate_min =		8000,
-	.rate_max =		192000,
+	.rate_max =		384000,
 	.channels_min =		1,
 	.channels_max =		32,
 	.buffer_bytes_max =	2 * 1024 * 1024,
